sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -87,6 +88,14 @@ export class ChangeFeedIteratorResponse<T> {
     readonly subStatusCode?: number;
 }
 
+// @public (undocumented)
+export enum ChangeFeedMode {
+    // (undocumented)
+    AllVersionsAndDeletes = "Full-Fidelity Feed",
```

Comment:
Can we follow consistent capitalization here like Full-Fidelity Feed, Incremental Feed